README.md

```diff
@@ -292,6 +292,18 @@ end
 config.add_observer 'ReputationChangeObserver'
 ```
 
+### I18n
+
+Merit uses default messages with I18n for notify alerts. To customize your app, you can set up your locale file:
+
+```yaml
+en:
+  merit:
+    granted_points: "granted %{points} points"
+    granted_badge: "granted %{badge_name} badge"
+    ...
```

Comment:
Given we only have three translations for the entire project, we can list them all here.